lib/models/analytic/aiuta_analytic_event_type.dart

```diff
@@ -0,0 +1,39 @@
+enum AiutaAnalyticEventType {
+  // Pages
+  // Onboarding
+  WelcomePageEntered,
+  HowItWorksPageEntered,
+  BestResultPageEntered,
+  ConsentPageEntered,
+  // Picker
+  ImagePickerPageEntered,
+  LoadingTryOnPageEntered,
+  // Result
+  ResultPageEntered,
+  // History
+  HistoryPageEntered,
+
+  // Actions
+  // Onboarding
+  WelcomeScreenStartButtonClicked,
+  ConsentGiven,
+  // Picker
+  TryOnStarted,
+  NewPhotoStarted,
+  NewPhotoTaken,
+  NewPhotoUploaded,
+  TryOnResultsError,
+  // Result
+  TryOnResultsShown,
+  FlowExit,
+  GeneratedImageShared,
+  ProductLike,
+  ProductAddToCart,
+  // Feedback
+  FeedbackPositive,
+  FeedbackNegative,
```

Comment:
params:
- feedback: String

Reply:
Let's add `option: Int` - to indicate index of selected option